server/db/mongodb.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+
+mongoose.connect('mongodb://127.0.0.1:27017/reviews');
+
+const reviewsSchema = new mongoose.Schema({
+  id: Number,
+  product_id: Number,
+  rating: Number,
+  summary: String,
+  recommend: Boolean,
+  body: String,
+  name: String,
+  email: String,
+  photos: {
+    id: Number,
+    url: String,
+  },
+  characteristics: {
+    id: Number,
+    product_id: Number,
+    characteristics_reviews: {
+      id: Number,
+      rating: Number,
+    },
+  },
+});
```

Comment:
In case you wanted to use MongoDB, don't forget to model your schema with `const Reviews = mongoose.model('Reviews', reviewsSchema)`. Just pointing it out, but I don't think you need it because you are doing Postgres.

Reply:
yeah I figured thank you :)